src/manager/agents/common-module/module/common_module.cpp

```diff
@@ -20,10 +22,13 @@ sc_result CommonModule::InitializeImpl()
     SC_LOG_ERROR("CommonModule is deactivated");
     return SC_RESULT_ERROR;
   }
-
+  ScMemoryContext * m_memoryCtx = new ScMemoryContext();
```

Comment:
```suggestion
  ScMemoryContext context;
```
имя не надо такое ужасное делать, это не member. И не думаю, что нужен указатель